src/main/webapp/resources/js/pages/analysis/components/Analysis.jsx

```diff
@@ -123,15 +156,15 @@ export default function Analysis() {
 
               <TabPane
```

Comment:
I am thinking that we might want each of these tabs to be lazy loaded, what do you think?

Reply:
Yup that's the plan!

Reply:
Added in dc1a014b789d7270dd33c9e420a21a122a5e796a